src/modules/ModuleGovernance/List/Wrap.vue

```diff
@@ -0,0 +1,129 @@
+<script setup lang="ts" name="ModuleGovernanceListWrap">
+import { useGovernanceStore } from '../store/governance'
```

Comment:
Maybe add governance store to auto-import?

Reply:
How does it work now? I tried, but I didn't figure out how to do it

Reply:
You can add the path here: 

https://github.com/soramitsu/klaytn-dex-frontend/blob/395e2109f3581998f755eaa756bf9838491c49ec/vite.config.ts#L77-L92

Reply:
Done